<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_1">
       <attribute name="title">
        <string>Fightercraft</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="2">
         <widget class="QPushButton" name="buttonViewPilot">
          <property name="text">
           <string>View Pilot</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QPushButton" name="buttonAdd_1">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="buttonEdit_1">
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QPushButton" name="buttonDel_1">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="buttonViewCopilot">
          <property name="text">
           <string>View Copilot</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="buttonViewModel">
          <property name="text">
           <string>View Model</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="6">
         <widget class="QTableView" name="tableView_1"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Pilots</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Dogtag</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Surname</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Race</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLineEdit" name="lineName"/>
        </item>
        <item row="6" column="0">
         <widget class="QDateEdit" name="dateEdit"/>
        </item>
        <item row="6" column="3">
         <widget class="QPushButton" name="buttonDel_2">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2" colspan="2">
         <widget class="QLineEdit" name="lineRace"/>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineSurname"/>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="lineRank"/>
        </item>
        <item row="0" column="0" colspan="4">
         <widget class="QTableView" name="tableView_2"/>
        </item>
        <item row="6" column="2">
         <widget class="QPushButton" name="buttonAdd_2">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="buttonViewAll_1">
          <property name="text">
           <string>View All</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLineEdit" name="lineDogtag"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Birth date</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Rank</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="labelFilter_1">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Models</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="3" column="1">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Fuel</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Type</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Class</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLineEdit" name="lineLowAtmoEngines"/>
        </item>
        <item row="6" column="3">
         <widget class="QPushButton" name="buttonAdd_3">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineFuel"/>
        </item>
        <item row="4" column="3">
         <widget class="QLineEdit" name="lineHighAtmoEngines"/>
        </item>
        <item row="8" column="3">
         <widget class="QPushButton" name="buttonDel_3">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLineEdit" name="lineModelName"/>
        </item>
        <item row="2" column="3">
         <widget class="QLineEdit" name="lineClass"/>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineType"/>
        </item>
        <item row="8" column="1">
         <widget class="QLineEdit" name="lineSecondaryWeapon"/>
        </item>
        <item row="8" column="0">
         <widget class="QLineEdit" name="linePrimaryWeapon"/>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="lineFTLDrive"/>
        </item>
        <item row="4" column="0">
         <widget class="QLineEdit" name="linePowercore"/>
        </item>
        <item row="0" column="0" colspan="5">
         <widget class="QTableView" name="tableView_3"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Powercore</string>
          </property>
         </widget>
        </item>
        <item row="8" column="4">
         <widget class="QPushButton" name="buttonViewAll_2">
          <property name="text">
           <string>View All</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>High atmo engines</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Low atmo engines</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>FTL drive</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Primary weapon</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Secondary weapon</string>
          </property>
         </widget>
        </item>
        <item row="7" column="4">
         <widget class="QLabel" name="labelFilter_2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QPushButton" name="buttonIdiotCounter">
          <property name="text">
           <string/>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QLCDNumber" name="lcdIdiotCounter"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>528</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
